docs/calicoctl/container.md

```diff
@@ -0,0 +1,180 @@
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a 
+full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl container' commands
+
+Run `calicoctl container --help` to display the following help menu for the 
+calicoctl container commands.
+
+```
+
+Usage:
+  calicoctl container add <CONTAINER> <IP> [--interface=<INTERFACE>]
+  calicoctl container remove <CONTAINER>
+  calicoctl container <CONTAINER> ip (add|remove) <IP> [--interface=<INTERFACE>]
+  calicoctl container <CONTAINER> endpoint show
+  calicoctl container <CONTAINER> profile (append|remove|set) [<PROFILES>...]
+
+Description:
+  Add or remove containers to calico networking, manage their IP addresses and profiles.
+  All these commands must be run on the host that contains the container.
+
+Options:
+  --interface=<INTERFACE>  The name to give to the interface in the container
+                           [default: eth1]
+
+```
+
+## calicoctl container commands
+
+
+### calicoctl container add <CONTAINER> <IP> 
+
+This command allows you to add a container into the Calico network. 
+
+This command is required for any container created using default Docker 
+networking to use Calico.  This command creates a network endpoint with the 
+given IP address on the container's Calico interface.  
+
+To configure networking policy on a container after it has been added to 
+Calico, create a profile using `calicoctl profile add` (see the 
+[`calicoctl profile` guide](./profile.md)) and set the profile on the container 
+using the `calicoctl container <CONTAINER> profile add <PROFILE>` command (see 
+below).
+
+This command must be run as root and must be run on the specific Calico node 
+that hosts the container.
+
+Command syntax:
+
+```
+calicoctl container add <CONTAINER> <IP> [--interface=<INTERFACE>]
+
+    <INTERFACE>: The name to give to the interface in the container
+                 [default: eth1]
+    <CONTAINER>: The name or ID of the container
+    <IP>: The IPv4 or IPv6 address to add.
+```
+
+Examples:
+
+```
+$ calicoctl container add test-container 192.10.0.3 --interface=eth1 
+IP 192.10.0.3 added to test-container
+```
+
+### calicoctl container remove <CONTAINER>
+
+This command allows you to remove a container from the Calico network.
+
+This command must be run as root and must be run on the specific Calico node 
+that hosts the container.
+
+Command syntax:
+
+```
+calicoctl container remove <CONTAINER>
+
+    <CONTAINER>: The name or ID of the container
+```
+
+Examples:
+
+```
+$ calicoctl container remove test-container
+Removed Calico interface from test-container
+```
+
+### calicoctl container <CONTAINER> ip add <IP> 
+
+This command allows you to add an IP address to a container that has already
+been configured to use Calico networking with the `calicoctl container add` 
+command (see above).
+
+This command must be run as root on the specific Calico node that hosts the 
+container.
+
```

Comment:
"If you specify the `--interface` flag, this interface must already exist within the container."